FormlyMapper/FormlyField.php

```diff
@@ -35,9 +35,15 @@ public function getFormlyFieldConfiguration()
      */
     protected function buildConfiguration()
     {
+        $this->formlyFieldConfiguration = [];
```

Comment:
Ok, so we discovered the reason for this is a really bad factory acting as a manager. We'll leave it like this for this PR, and then fix it in another, since changing the factory may require a bunch of changes.